cmake/Toolchains/aarch64-linux-gnu.cmake

```diff
@@ -9,7 +9,10 @@
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_PROCESSOR aarch64)
 
-if(MINGW OR CYGWIN OR WIN32)
+if(
+  MINGW
+  OR CYGWIN
+  OR WIN32)
```

Comment:
this is extremely ugly.

Reply:
I re-added the corresponding file from gnuradio and the result is almost the same but without the new line after `if(`. I'd assume this is intended.